pkg/appStore/AppStoreDeploymentService.go

```diff
@@ -343,12 +345,13 @@ func (impl InstalledAppServiceImpl) updateValuesYaml(environment *repository5.En
 		impl.logger.Errorw("error in marshaling", "err", err)
 		return err
 	}
-	valuesYaml := &util.ChartConfig{
+	gitOpsRepoName := impl.chartTemplateService.GetGitOpsRepoName(installedAppVersion.AppStoreApplicationVersion.AppStore.Name)
```

Comment:
I am ignoring reviewing this business logic as I don't have context of this story